src/widgets/widget_definitions.rs

```diff
@@ -319,6 +324,7 @@ fn build_gtk_box(bargs: &mut BuilderArgs) -> Result<gtk::Box> {
 }
 
 /// @widget label
+/// @desc A widget that displays a small to medium amount of text
```

Comment:
Maybe explain why this is relevant if you can just use inline text too
```suggestion
/// @desc A text widget giving you more control over how the text is displayed
```

Reply:
i copied that from the gnome docs. pls no bully 
![image](https://user-images.githubusercontent.com/30902201/97174278-c64fa980-1789-11eb-9f45-eb4d9b3e9518.png)